src/pages/patientView/mutation/PatientViewMutationsTab.tsx

```diff
@@ -46,6 +50,30 @@ export default class PatientViewMutationsTab extends React.Component<
     IPatientViewMutationsTabProps,
     {}
 > {
+    get showTimeline() {
+        return (
+            this.props.urlWrapper.query.genomicEvolutionSettings
+                .showTimeline === 'true'
+        );
+    }
+    set showTimeline(o: boolean) {
+        this.props.urlWrapper.updateURL({
+            genomicEvolutionSettings: Object.assign(
```

Comment:
i feel like there's got to a nicer way to do this.  it's basically the old immutable issue.  I think m.westtrate has some library which handles stuff like this.  we should look into it.  not necessary for this PR though

Reply:
ill do this in a different PR